R/allowed_tags.R

```diff
@@ -0,0 +1,4 @@
+#' This is a vector of all the allowed blog post tags. It is stored in a separate file so that it can be referenced
+#' by `create_blogpost.R` and `check_post_tags.R`.
+
+allowed_tags <<- c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")
```

Comment:
Would this also work without the superassigner?

Reply:
Maybe. Originally I didn't pass the allowed tags into the check function which is why I had them superassigned. But let me try remove in the next update when I remove the linebreak from the PR guidance.

Reply:
Update - still works :)